python/test/t_ParametricFunction_std.py

```diff
@@ -37,3 +38,50 @@
 
 print(modelFunc.gradient([s[0], r, c]).transpose())
 print(mycf.parameterGradient(s).transpose())
+
+# Check if parametric functions and memoize functions work well together
+
+n_calls = 0
+def py_f(X):
+    global n_calls
+    n_calls += 1
+    return X
+
+ot_f = ot.MemoizeFunction(ot.PythonFunction(3, 3, py_f))
+param_f = ot.ParametricFunction(ot_f, [0, 1], [1.0, 2.0])
+
+x = [3.0]
+y = [1.5]
+n_calls_0 = ot_f.getCallsNumber()
+par_grad = param_f.parameterGradient(x)
+n_calls_1 = ot_f.getCallsNumber()
+assert_almost_equal(n_calls_1 - n_calls_0, 4, 0.0, 0.0)
+assert_almost_equal(n_calls, 4, 0.0, 0.0)
```

Comment:
it makes no sense to use assert_almost_equal for ints, just use `assert n_calls == 4, "message"`